cves/CVE-2017-11665.yml

```diff
@@ -172,10 +171,10 @@ discovered:
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked. Thus, 'answer' should always have some
     explanation.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: "found by JunDong Xie of Ant-financial Light-Year Security Lab"
+  automated: false
+  contest: false
+  developer: true
```

Comment:
How do you know that the finder was a developer on the project? This sounds like it came from a security lab, which sounds more likely to be external.